pkgs/applications/graphics/krita/generic.nix

```diff
@@ -60,6 +56,8 @@ mkDerivation rec {
   cmakeFlags = [
     "-DPYQT5_SIP_DIR=${python3Packages.pyqt5}/${python3Packages.python.sitePackages}/PyQt5/bindings"
     "-DPYQT_SIP_DIR_OVERRIDE=${python3Packages.pyqt5}/${python3Packages.python.sitePackages}/PyQt5/bindings"
+    "-DBUILD_KRITA_QT_DESIGNER_PLUGINS=ON"
+    "-DBUILD_TESTING=OFF"
```

Comment:
I think it is important to make sure the tests keep passing, especially with all the non-standard build setup in nixpkgs. Any reason why we disable the test here?

Reply:
Build went through ok on my builder.